### PR TITLE
add envTags to sentry error reporting

### PR DESCRIPTION
**Pull request by christopherriley**
##### WHAT

add envTags to js-logger
allows us to tag sentry errors with things like cluster name, build, etc (like supernova logger)

##### WHO

@rzlee @gerad

### CODE REVIEW

File: mergeTags.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        }
+    });
+    return target;
+}
+
+function mergeTags(tags1, tags2) {
+  return extend({}, tags1, tags2); 
+}
+
+module.exports = mergeTags;
```
Comment (gerad) [inline mergeTags.js]: :-1: just use `Object.assign()` which does the exact same thing and is built-in

**Comment by gerad** [comment]
:-1: rewrite using built-in sentry functionality: https://docs.getsentry.com/hosted/clients/node/usage/

usage sentry documentation 2016-03-02 04-19-04

**Comment by gerad** [comment]
Also, please update the usage section of the README

**Comment by christopherriley** [comment]
> rewrite using built-in sentry functionality

done.  thanks, should have read the docs.


> just use Object.assign() which does the exact same thing and is built-in

removed it, per previous review comment


> update the usage section of the README

done

File: logger.js [hunk 2]
```diff
@@ -4,8 +4,9 @@
 var raven = require("raven");
 
 class Logger {
-  constructor(serviceName) {
+  constructor(serviceName, envTags) {
     this.serviceName = serviceName;
+    this.envTags = envTags;
```
Comment (gerad) [inline logger.js]: rm

Comment (christopherriley) [inline logger.js]: > rm this.envTags = envTags;

done

**Pull request merged**